wiki/BBCode/id.md

```diff
@@ -1,396 +1,302 @@
-
-- Tombol khusus: N/A
-- Catatan:
-  - Ketahuilah ini tidak akan memberikan hasil yang sama kepada semua orang.
-    - Beberapa hasil pencarian mungkin tersembunyi karena masalah bahasa/lokasi.
-- Sintaks:
-  ```
-  [lucky]...[/lucky]
-  ```
-
-### List
-
-Mengubah teks dalam bentuk daftar.
-
-di dalam BBCode, terdapat dua bagian untuk membuat daftar: wrapper dan bullets. Dimana bullets diletakkan di dalam wrapper.
-Bullets ialah simbol yang berbentuk sebuah peluru, sedangkan Wrapper ialah membungkusi/menyoroti teks yang sudah memiliki masing-masing Bullets dan membungkusnya menggunakan tag list.
-
-- Tombol khusus:
-  - Bulleted list: ![tombol list](img/list.png)
-  - Numbered list: ![tombol nomor list](img/list-numbered.png)
-  - List bullet: N/A
-  - List type: N/A
-- Catatan:
-  - Semua bullets harus dikelilingi oleh tag list.
-  - Berikut adalah argumen yang benar dari `TYPE` bullets:
-    - *(empty)* - bulleted
-    - `1` - numbered
-    - `a` - lettered (lowercased)
-    - `A` - lettered (upper-cased)
-    - `i` - roman numeral (lowercased)
-    - `I` - roman numeral (upper-cased)
-  - Secara standar, bulleted list menggunakan square bullets.
-  - List di dalam list bisa menyebabkan kesalahan pada teks tersebut.
-- Sintaks:
-  - Dimana `TYPE` adalah salah satu tipe daftar diatas.
-  - Jika `TYPE` tidak digunakan, bullets standar akan digunakan sebagai gantinya.
-  ```
-  [list=TYPE]
-  [*]...
-  [/list]
-  ```
-
-### Image
-
-**Menampilkan gambar dari sumber gambar online**.
-
-Gambar dapat diperoleh dari sumber manapun, selama gambar tersedia dan memiliki url.
-
-**Jangan link gambar langsung dari lokal pc!** Menggunakan `C:\Users\Name\Pictures\image.jpg` **tidak akan bisa digunakan**.
-
-Tolong unggah gambar ke website yang terpercaya seperti [imgur](https://imgur.com) atau [puush](https://puush.me). Setelah gambar diunggah berhasil, salin dan tempel tautan yang berisikan gambar. Tidak semua website memberikan tautan langsung pada gambar yang diunggah (atau biasa disebut *hotlinks*). Situs berbagi gambar, seperti ada yang diatas, sangat diperbolehkan menggunakan tautan langsung pada gambar karena mereka adalah situs berbagi gambar.
-
-- Tombol khusus: ![Tombol Image](img/image.png)
-- Catatan:
-  - Jika Anda memiliki banyak gambar atau gambar berukuran besar, sangat direkomendasikan meletakkannya di dalam [Box](#box).
-- Sintaks:
-  - Dimana `LINK` adalah tautan langsung dari gambar tersebut.
-  ```
-  [img]LINK[/img]
-  ```
+### Images
+
+```
+[img]ADDRESS[/img]
+```
+
+Tag `[img]` atau *gambar* digunakan untuk menyematkan tautan gambar berekstensi dari online ke dalam postingan forum osu!. Untuk menggunakan tag, pengguna harus menempelkan alamat gambar langsung (diwakili oleh argumen `ADDRESS` di atas) yang bersumber dari situs web. Jalur file lokal. Menggunakan tautan langsung dari komputer (*offline*), (misalnya, `C:\Users\Name\Pictures\image.jpg`), **tidak akan berfungsi**.
```

Comment:
```suggestion
Tag `[img]` atau *gambar* digunakan untuk menyematkan tautan gambar berekstensi dari online ke dalam postingan forum osu!. Untuk menggunakan tag, pengguna harus menempelkan alamat gambar langsung (diwakili oleh argumen `ADDRESS` di atas) yang bersumber dari situs web. Menggunakan tautan langsung dari komputer (*offline*), (misalnya, `C:\Users\Name\Pictures\image.jpg`), **tidak akan berfungsi**.
```
sepertinya kelebihan kalimat ya?

Reply:
actually, this is intended, seems the official didn't telling which address has to use.